modules/prometheus-exporter_oracledb/conf/readme.yaml

```diff
@@ -0,0 +1,83 @@
+documentations:
+  - name: Prometheus Exporter for oracledb
+    url: https://github.com/iamseth/oracledb_exporter
+
+source_doc: |
+  The detectors of this module uses defaults metrics from the [oracledb exporter] (https://github.com/iamseth/oracledb_exporter) and optionnaly custom metrics from custom templates which have to be specified during the deployment.  
+  Check its documentation to install and configure it appropriately with your Oracle database host.
+
+  by default in this module, only the metric oracledb_up is used with the default.metrics related to the [oracledb exporter]. 
+
+  ### Examples
+
+  Here is a sample configuration fragment for the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) agent using
+  the [prometheusexec receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusexecreceiver).
+  
+  In this example we have an oracle instance which is running in Multitenant mode, means that we have one Container Database and one pluggable database. 
+  
+  important : Both needs a dedicated prometheus port to upload metrics.
+  
+  {PATH_TO_ORACLEDB_BIN} = directory related to the oracledb_exporter binary which is used,
+  {PATH_TO_ORACLEDB_TEMPLATE} = directory related to the templates which are used (default & custom)
+  {ORACLE_HOME}= Oracle Home related to rdbms software
+  {TNS_PORT} = Oracle listener port on which the DB is listening.
+  {CONTAINER_DB} = name of the container DB (instance name)
+  {PLUGGABLE_DB} = name of the pluggable Database
+  {HOSTNAME} = name of the DB server host
+  
+  ```yaml
```

Comment:
please append `$` at the start of each variable use and remove `{` `}` for what you did not define a variable (e.g. `{USER}` -> `myuser`)

Reply:
done